autoload/neomake/utils.vim

```diff
@@ -409,3 +409,22 @@ endfunction
 function! neomake#utils#path_sep() abort
     return neomake#utils#IsRunningWindows() ? ';' : ':'
 endfunction
+
+" Find a file by going up the directories form the start directory
+" and performing glob search for the file.
+function! neomake#utils#FindGlobFile(startDir, file)
+    let currDir = a:startDir
+    let fileFound = ""
+
+    while empty(fileFound)
+        let fileFound = globpath(currDir , a:file, 1)
+        let lastFolder = currDir
+        let currDir = fnamemodify(currDir, ':h')
+
+        if currDir ==# lastFolder
+            break
+        endif
+    endwhile
+
+    return fileFound
+endfunction
```

Comment:
Yeah I based on this one, it does a bunch of stuff that I do not know if they are needed, I tried to keep it simple, but you are the maintainer what you think ?